src/main/java/net/minecraftforge/common/extensions/IForgeEntity.java

```diff
@@ -200,4 +202,19 @@ default boolean isMultipartEntity()
     {
         return null;
     }
+
+    /**
+     * Return the height in blocks the Entity can step up without needing to jump
+     * @return The sum of vanilla's {@code maxUpStep} field and the current value
+     *  of the {@link net.minecraftforge.common.ForgeMod#STEP_HEIGHT} attribute,
+     *  (if this Entity is a {@link LivingEntity}).
```

Comment:
Suggestion for improving the javadocs:
```suggestion
     * {@return the height in blocks that the Entity can step up without needing to jump} This is the sum of 
     * the entity's {@link Entity#maxUpStep} and the value of the of the 
     * {@link net.minecraftforge.common.ForgeMod#STEP_HEIGHT} attribute (if this is a {@link LivingEntity}).
```